CSS/firstjump.css

```diff
@@ -0,0 +1,180 @@
+body {
+  background-color: white;
+  padding: 0px;
+  margin: 0px;
+  width: 100%;
+}
+
+/* Font section */
+
+.site_font {
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.site_font.bold {
+  font-weight: 700;
+}
+
+.site_font.light {
+  font-weight: 300;
+}
+
+.site_font.regular {
+  font-weight: 400;
+}
+
+.site_alt_font_one {
+  font-family: cursive;
+  font-size: 24px;
+}
+
+.site_alt_font_two {
+  font-size: 24px;
+}
+
+/* Banner */
+
+#bannertitle {
+  font-size: 7vw;
+  text-align: center;
+  color: white;
+  background-color: #4282d6;
+}
+
+.banner_title {
+  margin-top: 0%;
+  background-color: #4282d6;
+  padding-top: 10%;
+}
+
+#bannerImageContainer {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+  background-color: #4282d6;
+}
+
+#bannerClouds {
+  width: 100%;
+  margin-top: 30%;
+  background-size: cover;
+  overflow: hidden;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  position: relative;
+}
+
+#bannerSkydiver {
+  width: 80%;
+  position: absolute;
+  margin-left: 12%;
+  margin-bottom: 5%;
+}
+
+#infoSection {
+  margin-top: -2%;
+  background-color: white;
+  width: 100%;
+  height: 800px;
+  /* display: flow-root; */
+  margin-left: 5%;
+}
+
+#parachute {
+  float: right;
+  width: 40%;
+  display: flow-root;
+  margin-right: 0%;
+}
+#section {
+  margin-top: 50%;
+  margin-left: 5%;
+}
+
+#formHeader {
+  text-align: center;
+}
+
+#form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 60%;
+  margin-left: 20%;
+  margin-top: 10%;
+  font-size: 2vw;
+}
+
+#submit {
+  width: 30%;
+  height: 40px;
+}
+
+#lowerLogo {
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+  margin-top: 10%;
+}
+
+#uspa {
+  width: 10%;
+  margin-bottom: 5%;
+}
+
+/* Footer Section */
+
+#footerContainer {
+  width: 100%;
+  background-color: rgb(248, 248, 248);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-size: 6vw;
+}
+#footerLeft {
+  width: 20%;
+  min-width: 20%;
+}
+
+#footerLogo {
+  display: flex;
+  flex-direction: row;
+  cursor: pointer;
+}
+
+#footerhelmet {
+  width: 30%;
+  height: 90%;
+  margin-left: 5%;
+  margin-top: 2%;
+}
+
+#footerRight {
+  width: 100%;
+  margin-left: 20%;
+  display: flex;
+  justify-content: flex-end;
+}
+
+#socialLogos {
+  cursor: pointer;
+  width: 22%;
+}
+
+#socialImage {
+  width: 33%;
+  height: 60%;
+  margin-left: 2%;
+  margin-top: 2%;
+}
+
+p5 {
```

Comment:
This is incorrect. Unlike the H1, H2, H3 elements there is no p1, p2, p3 etc. Just the one p element. You could make this into a class though. Youll need to pull out any references to p5 elements in your code and replace them with just p elements. The browser is helping you out here by assuming thats a typo and fixing it for you.